package.json

```diff
@@ -50,6 +50,7 @@
   },
   "dependencies": {
     "chartjs-color": "^2.1.0",
+    "gulp-uglifyes": "0.1.2",
```

Comment:
not sure how I feel about adding a new package for syntax purposes and 8 new lines of code